test/libyul/yulSyntaxTests/invalid/unicode_string_direction_override.sol

```diff
@@ -3,4 +3,4 @@
     let s := unicode"underflow ‬";
 }
 // ----
-// ParserError 1465: (35-47): Illegal token: Invalid character in string.
+// ParserError 1465: (35-47): Illegal token: Invalid character in string. If you want to use unicode characters, use a unicode"..." string literal.
```

Comment:
The message is now wrong here. Note that this **is** a `unicode` string. We just disallow some BiDi characters.

So we need to make the message more specific or start issuing a different error in this case.